waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/PanopticOperationHandler.java

```diff
@@ -79,6 +80,7 @@ protected List<String> getAllDatabases(
     List<String> result = getPanopticOperationExecutor()
         .executeRequests(allRequests, GET_DATABASES_TIMEOUT, "Can't fetch databases by pattern: {}");
     log.info("All Databases Result={}", result);
+    result = result.stream().distinct().collect(Collectors.toList());
```

Comment:
Please remove this and write a test to reproduce this erorr. https://github.com/ExpediaGroup/waggle-dance/blob/hive-3.x/waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java

Best to raise a separate PR for this issue

Reply:
Thank you for your suggestion. I have removed the commit.
I will try to reproduce the issue and raise a separate PR for this issue within the next week.